sites/docs/src/content/docs/tutorials/storage_utilization/managing_work_directory_growth.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Managing Nextflow work directory growth"
+subtitle: "A guide for efficient storage utilization"
+---
+
+The management of intermediate files generated during Nextflow pipeline execution is a challenge for some workflows. As pipelines increase in complexity and scale, work directories can rapidly consume available storage, potentially leading to pipeline failures. This guide summarizes strategies for managing work directory growth while maintaining pipeline reproducibility and debugging capabilities.
+
+## Work directory accumulation
+
+The Nextflow work directory serves as an important component of the execution model, providing caching capabilities and resume functionality. During pipeline execution, Nextflow creates unique subdirectories for each task (e.g., `work/3f/70944c7a549b6221e1ccc7b4b21b62`) containing:
+
+- Symbolic links to input files
+- Intermediate output files
+- Command scripts and execution logs
+- Temporary files created during tasks
+
+In production environments processing large-scale genomic datasets, work directories can expand rapidly.
+
+## Storage management options
+
+The following sections describe ways to manage the storage of files.
+
+### Selective post-execution cleanup
+
+Nextflow's built-in `clean` command enables targeted removal of work directories. The following command removes directories from the most recent execution:
+
+```bash
+nextflow clean -f -before $(nextflow log -q | tail -n 1)
+```
+
+Command components:
+
+- `nextflow log -q{:bash}`: Retrieves execution history without headers
+- `tail -n 1{:bash}`: Isolates the most recent execution identifier
+- `-before{:bash}`: Specifies cleanup of executions preceding the specified run
+- `-f{:bash}`: Executes deletion without confirmation
+
+For verification, perform a dry run by omitting `-f{:bash}`:
```

Comment:
I think you need `-n` if you want to do a dry run. One of `-f` or `-n` are required.

Reply:
For verification, perform a dry run using the `-n{:bash}` option